### DISCUSSION TITLE
Makefile dependencies

### DISCUSSION THREAD

**Discussion by psumesh** [cocotb/cocotb#2807] [Q&A] [answered]
Hi,

Can cocotb environment run without make file?

**Comment by ktbarrett** [comment, marked as answer]
Yes. See our documentation on [running cocotb in a different flow](https://docs.cocotb.org/en/stable/custom_flows.html). Also see the [cocotb-test](https://github.com/themperek/cocotb-test) project for running in Python. And finally, for the 1.7 release we hope to have experimental support for a new [Python runner](https://github.com/cocotb/cocotb/pull/2634).
